egs/apiai_decode/s5/download-model.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Downlaods Api.ai chain model into exp/api.ai-model/ (will replace one if exists)
+
+DOWNLOAD_URL="https://api.ai/downloads/api.ai-kaldi-asr-model.zip"
+
```

Comment:
OK, well eventually it will make sense to update the README to mention
this, but I guess we can do that later on.


On Mon, Apr 4, 2016 at 6:38 PM, Ilya Platonova <notifications@github.com>
wrote:

> In egs/apiai_decode/s5/download-model.sh
> <https://github.com/kaldi-asr/kaldi/pull/654#discussion_r58461119>:
>
> > @@ -0,0 +1,24 @@
> > +#!/bin/bash
> > +# Downlaods Api.ai chain model into exp/api.ai-model/ (will replace one if exists)
> > +
> > +DOWNLOAD_URL="https://api.ai/downloads/api.ai-kaldi-asr-model.zip"
> > +
>
> If we will go into versioning, then
> https://api.ai/downloads/api.ai-kaldi-asr-model.zip will be latest
> version (link), and other versions will have numbers at the end.
>
> Also we will track everything here
> https://github.com/api-ai/api-ai-english-asr-model
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/654/files/052115e3005f9ca2b86ebb8f73a81b3b8046ebaa#r58461119>
>